src/code_800AF9B0.c

```diff
@@ -4,10 +4,26 @@
 #include "global.h"
 #include "common_structs.h"
 
-u32 func_802A7B70(u16 x);
+void func_800B44BC(void);
+void func_800B4670(void);
+void func_800B4728(s32 arg0);
+void func_800B4820(void);
+s32  func_800B492C(void);
+s32  func_800B49E4(void);
+void func_800B559C(s32 arg0);
+void func_800B5948(void);
+u32  func_802A7B70(u16 x);
+
+extern u8 D_8018ED10[4]; // Direct reference to the grandPrixPoints section of save data
+extern u8 D_8018ED11;
+extern u8 D_8018ED12;
+extern u8 D_8018ED13;
+extern s8 D_8018ED16;
+extern s8 D_8018ED17;
```

Comment:
I see there's an include/variables.h file... Should these declarations be moved into there and this file includes it?

Reply:
I tend to only put named variables in that file, but I don't think there's any actual restrictions on that. If you find it useful to put it in there I'd vote for just doing it

Reply:
The backstory here is. Somebody decomped a single file (for which I very much appreciate). And they did this for it.
I haven't paid much attention to it. If you find it useful, do whatever you want with it.

Reply:
ok, I'll leave these here then